apps/frontend/src/components/system-metrics-chart.tsx

```
@@ -0,0 +1,104 @@
+            <div className="w-3 h-3 bg-green-500 rounded-full"></div>
+            <span>Memory</span>
+          </div>
+          <div className="flex items-center space-x-1">
+            <div className="w-3 h-3 bg-purple-500 rounded-full"></div>
+            <span>Network</span>
+          </div>
+        </div>
+      </div>
+
+      <div className="h-64 w-full">
+        <svg viewBox="0 0 800 200" className="w-full h-full">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)